db/generateFakerData.js

```diff
@@ -0,0 +1,41 @@
+const faker = require('faker');
+const moment = require('moment');
+
+const getRandomNum = function () {
+  const min = 1;
+  const max = 6;
+  return Math.floor(Math.random() * ((max - min) + 1)) + min;
+};
+
+const generateReviews = function () {
+  const reviews = [];
+  for (let i = 0; i < getRandomNum(); i += 1) {
```

Comment:
You can use faker number generator here instead of creating an entire function.

Reply:
Of course! Changed.